classesExamples/PassionFruit.py

```diff
@@ -0,0 +1,20 @@
+from classesExamples.Fruit import Fruit
+
+
+class PassionFruit(Fruit):
+
+    def nutrition(self):
+        print('this is the more delicious, but not the most nutritive one')
```

Comment:
❤️I love the naming and these fruity examples 🍑

Reply:
:D